src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
-
+import { Movie } from '../../types/movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
+    {/* eslint-disable-next-line */}
```

Comment:
Do we really need to disable linter here ?

Reply:
It was Vitalyk's recommendation due to the error I had spoken to you.